garageScript/c0d3-app
# Add DOJO exercises nav card

This pull request adds a simple nav card to the exercises page.

![exercises-nav](https://user-images.githubusercontent.com/7637655/189365193-005c4c5f-1f03-4b2d-8ef6-34f71f0facd4.png)


How to test:
* Go to /exercises/js0
* Check that the nav card is there
* Check that the nav card links work

This pull request is a part of https://github.com/garageScript/c0d3-app/issues/2253
